en/deploy-tidb-from-kubernetes-gke.md

```diff
@@ -98,25 +98,17 @@ If you see `Ready` for all nodes, congratulations! You've set up your first Kube
 TiDB Operator uses [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` CRD.
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.2/manifests/crd.yaml && \
 kubectl get crd tidbclusters.pingcap.com
 ```
 
 After the `TidbCluster` CRD is created, install TiDB Operator in your Kubernetes cluster.
 
-1. Install TiDB Operator:
-
-    ```shell
-    kubectl create namespace tidb-admin
-    helm install --namespace tidb-admin tidb-operator pingcap/tidb-operator --version v1.1.0
-    kubectl get po -n tidb-admin -l app.kubernetes.io/name=tidb-operator
-    ```
-
-2. Create the `pd-ssd` StorageClass:
-
-    ``` shell
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/persistent-disk.yaml
-    ```
+```shell
+kubectl create namespace tidb-admin
+helm install --namespace tidb-admin tidb-operator pingcap/tidb-operator --version v1.1.2
+kubectl get po -n tidb-admin -l app.kubernetes.io/name=tidb-operator
+```
```

Comment:
no need to install `pd-ssd` storage class anymore, because we use the default `gce-pd` storage class of GKE cluster in our examples, e.g.

```
$ kubectl get sc
NAME                 PROVISIONER            AGE
standard (default)   kubernetes.io/gce-pd   17m
```